code-challenges.test.js

```diff
@@ -13,61 +18,112 @@
+//create array and push into it
+// returns array of the length of the number inputed.
+// 
 
-const fibLength1 = 6
-// Expected output: [1, 1, 2, 3, 5, 8]
+// b) Create the function that makes the test pass.
+const greaterThanTwo = (num) => {
+  let array = [1, 1]
+  for(let i = 2; i < num; i++) {
+    array[i-2] + array[i-1] 
+    array.push(array[i-2] + array[i-1])
+  }
+  return array
 
-const fibLength2 = 10
-// Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
+}
 
-// b) Create the function that makes the test pass.
 
-// Pseudo code:
 
 // --------------------2) Create a function that takes in an object and returns an array of the values sorted from least to greatest.
 // Hint: Check out this resource: Object.values() https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Object/values
 
 // a) Create a test with expect statements for each of the variables provided.
+// Pseudo code: 
+//create a function named sortedArray that takes in an object as parameter
+//use the Object.values and sort to get the numbers from small to biggest
+describe("sortedArray", () => {
+  it("Create a function that takes in an object and returns an array of the values sorted from least to greatest", () => {
+    const studyMinutesWeek1 = {
+      sunday: 90,
+      monday: 30,
+      tuesday: 20,
+      wednesday: 15,
+      thursday: 30,
+      friday: 15,
+      saturday: 60
+    }
+    // Expected output: [15, 15, 20, 30, 30, 60, 90]
+    const studyMinutesWeek2 = {
+      sunday: 100,
+      monday: 10,
+      tuesday: 45,
+      wednesday: 60,
+      thursday: 20,
+      friday: 15,
+      saturday: 65
+    }
+    // Expected output: [10, 15, 20, 45, 60, 65, 100]
+    expect(sortedArray(studyMinutesWeek1)).toEqual([15, 15, 20, 30, 30, 60, 90])
+    expect(sortedArray(studyMinutesWeek2)).toEqual([10, 15, 20, 45, 60, 65, 100])
+  })
+})
 
-const studyMinutesWeek1 = {
-  sunday: 90,
-  monday: 30,
-  tuesday: 20,
-  wednesday: 15,
-  thursday: 30,
-  friday: 15,
-  saturday: 60
-}
-// Expected output: [15, 15, 20, 30, 30, 60, 90]
-
-const studyMinutesWeek2 = {
-  sunday: 100,
-  monday: 10,
-  tuesday: 45,
-  wednesday: 60,
-  thursday: 20,
-  friday: 15,
-  saturday: 65
+// b) Create the function that makes the test pass.
+const sortedArray = (object) => {
+  return Object.values(object).sort((a, b) => a - b)
 }
-// Expected output: [10, 15, 20, 45, 60, 65, 100]
+  
+
+
 
-// b) Create the function that makes the test pass.
 
-// Pseudo code:
 
 // --------------------3) Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
 
 // a) Create a test with expect statements for each of the variables provided.
-
+// Pseudo code: 
+//create a function called arraySum that takes in an array 
+//create a for loop to iterate the length of the array
+//create an empty array and use .push to push new sum into it
+// return the new array
```

Comment:
I would include that you made a variable called sum, and what it is doing in this function and in each iteration